contrib/windows/Vagrantfile

```diff
@@ -1,16 +1,33 @@
 # Vagrantfile for building Windows Julia via MSYS2 or Cygwin
 
+$script_cygwin = <<SCRIPT
+# change the following to 32 for 32 bit Julia
+$bits = "64"
+$arch = "x86_$bits".Replace("x86_32", "i686")
+$setup = "setup-$arch.exe".Replace("i686", "x86")
+
+mkdir -Force C:\\cygwin$bits | Out-Null
```

Comment:
yeah. I get annoyed when scripts stash things under C:\, so it would be nice to have an easy way to change. not a huge deal though.